Formula/olsrd.rb

```diff
@@ -62,6 +96,6 @@ def startup_plist
   end
 
   test do
-    assert_match version.to_s, pipe_output("#{sbin}/olsrd")
+    assert_match version.to_s, pipe_output("#{bin}/olsrd")
```

Comment:
Is there a reason for changing this. I don't think we have a policy to avoid `sbin`.

Reply:
Let me see if I can change `SBINDIR` to point to `sbin` instead of `bin`.